taegis-xdr/info.json

```diff
@@ -582,193 +998,124 @@
       ]
     },
     {
-      "operation": "isolate_assets",
-      "title": "Isolate Assets",
-      "annotation": "isolate_assets",
-      "description": "Isolates the asset based on the Asset ID and the isolation reason specified.",
+      "operation": "get_endpoint",
+      "title": "Get Endpoint",
+      "annotation": "get_endpoint",
+      "description": "Retrieves a RedCloak endpoint info based on the endpoint ID specified.",
       "category": "investigation",
       "is_config_required": true,
       "visible": true,
       "enabled": true,
       "output_schema": {
         "TaegisXDR": {
-          "AssetIsolation": {
-            "id": ""
+          "assetEndpointInfo": {
+            "hostId": "",
+            "hostName": "",
+            "actualIsolationStatus": "",
+            "allowedDomain": "",
+            "desiredIsolationStatus": "",
+            "firstConnectTime": "",
+            "moduleHealth": {
+              "enabled": "",
+              "lastRunningTime": "",
+              "moduleDisplayName": ""
+            },
+            "lastConnectAddress": "",
+            "lastConnectTime": "",
+            "sensorVersion": ""
           }
         }
       },
       "parameters": [
         {
           "name": "id",
-          "title": "Asset ID",
-          "type": "text",
-          "editable": true,
-          "visible": true,
-          "required": true,
-          "tooltip": "Specify the ID of the Asset which you want to isolate.",
-          "description": "Specify the ID of the Asset which you want to isolate."
-        },
-        {
-          "name": "reason",
-          "title": "Reason of Isolation",
+          "title": "Endpoint ID",
           "type": "text",
           "editable": true,
           "visible": true,
           "required": true,
-          "tooltip": "Specify a reason for isolation of the specified asset.",
-          "description": "Specify a reason for isolation of the specified asset."
+          "tooltip": "Specify the endpoint ID for which to fetch SecureWorks RedCloak\u2122 endpoint information.",
+          "description": "Specify the endpoint ID for which to fetch SecureWorks RedCloak\u2122 endpoint information."
         }
       ]
     },
     {
-      "operation": "update_alert_status",
-      "title": "Update Alert Status",
-      "annotation": "update_alert_status",
-      "description": "Adds a resolution status or modifies an existing resolution status based on the alert IDs and other input parameters specified.",
+      "operation": "get_playbook_execution",
+      "title": "Get Playbook Execution",
+      "annotation": "get_investigations_alerts",
```

Comment:
Please check the annotation I believe its wrong

Reply:
Resolved